content/en/docs/reference/using-api/_index.md

```diff
@@ -50,27 +50,30 @@ Here's a summary of each level:
 
 - Alpha:
   - The version names contain `alpha` (for example, `v1alpha1`).
-  - The software may contain bugs. Enabling a feature may expose bugs. A
-    feature may be disabled by default.
-  - The support for a feature may be dropped at any time without notice.
+  - Alpha API versions are disabled by default.
```

Comment:
```suggestion
- For built-in APIs, alpha versions are disabled by default. You must enable that alpha
  API group in your cluster before you can use it.
```
Is this always true? Any exceptions?

Reply:
For completeness, maybe we should make it clear what versioning looks like for alpha CRDs. I presume this is “if you don't want it, don't deploy it”.
Maybe also that an expectation that projects _within Kubernetes_ don't ship Helm / Terraform / Kustomize / whatever code with the alpha CRDs enabled? Perhaps a loose conventions for chart / module / etc to take a boolean that enables the alpha APIs, even before there are any beta or GA APIs to enable by default?

Reply:
updated to clarify built-in alpha APIs... I'd rather not tackle the CRD lifecycle in this PR